src/Pages/Main/Modal/Calendar.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components/macro';
+
+import React from 'react';
+
+export default function Calendar(props) {
+  return (
+    <div>
+      <CloseButton onClick={props.toClose} />
```

Comment:
```suggestion
export default function Calendar({ toClose }) {
  return (
    <div>
      <CloseButton onClick={toClose} />
```
함수 파라미터 자리에서 직접 구조분해 할당 가능할 거 같네요!

Reply:
네 수정하겠습니다